Steamfitter.Api/Services/TaskExecutionService.cs

```diff
@@ -537,6 +549,66 @@ private STT.Task<string> RunTask(TaskEntity taskToExecute, ResultEntity resultEn
             return task;
         }
 
+        private async STT.Task<string> HttpTaskTask(TaskEntity taskToExecute)
+        {
+            HttpResponseMessage response;
+            using (var scope = _scopeFactory.CreateScope())
+            {
+                var tokenResponse = await ApiClientsExtensions.GetToken(scope);
+                var actionParameters = JsonSerializer.Deserialize<HttpInputString>(taskToExecute.InputString);
+                var url = actionParameters.Url;
+                if (url.Contains("?"))
+                {
+                    url += $"&asuser={taskToExecute.UserId.ToString()}" ;
```

Comment:
Is the plan to add support for this query parameter to other apps? I have mixed feelings about that.

Reply:
I'm not sure what it would be used for, but this let's the other app know what user kicked off the task, without having to query steamfitter.  I'll just take it out for now.